Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/BoundsControl.cs

```diff
@@ -965,11 +971,6 @@ private void InitializeRigRoot()
             var rigRootObj = new GameObject(rigRootName);
             rigRoot = rigRootObj.transform;
             rigRoot.parent = Target.transform;
-
-            var pH = rigRootObj.AddComponent<PointerHandler>();
```

Comment:
Out of curiosity, why do these need to be removed?

Reply:
BoundsControl was changed just as a demonstration of how to use the registry. With the setup that we have in the examples or tests BoundsControl could as a possibility handle pointer events on its own, receiving only bubble up events. The necessity for a PointerHandler wouldnt be necessary. But this changes if user adds another pointer handler to the same object also handling bubble up. (Like a manipulion handler using children panels as manipulation handles). In this case the only options would be adding multiple PointerHandler children to the object or checking target object when handling bubbleup event)

Reply:
it makes sense to revert and keep BoundsControl spawning a PointerHandler, as it was.